src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -1195,5 +1197,1805 @@ static void Main()
+                {
+                    builder.Append(@"
+            _builder.Append("",alignment:"");
+            _builder.Append(alignment);");
+                }
+
+                if (includeFormat)
+                {
+                    builder.Append(@"
+            _builder.Append("":format:"");
+            _builder.Append(format);");
+                }
+
+                builder.Append(@"
+            _builder.AppendLine();");
+
+                if (useBoolReturns)
+                {
+                    builder.Append($@"
+            return {returnExpression ?? "true"};");
+                }
+
+                builder.AppendLine(@"
+        }");
+            }
+
+            void appendSignature(string nameAndParams)
+            {
+                builder.Append(@$"
+        public {(useBoolReturns ? "bool" : "void")} {nameAndParams}");
+            }
+
+            void appendNonDefaultVariantsWithGenericAndType(string type, string generic, bool isSpan = false)
+            {
+                appendSignature($"AppendFormatted{generic}({type} value)");
+                appendBody(includeValue: true, includeAlignment: false, includeFormat: false, isSpan);
+                appendSignature($"AppendFormatted{generic}({type} value, int alignment)");
+                appendBody(includeValue: true, includeAlignment: true, includeFormat: false, isSpan);
+                appendSignature($"AppendFormatted{generic}({type} value, string format)");
+                appendBody(includeValue: true, includeAlignment: false, includeFormat: true, isSpan);
+                appendSignature($"AppendFormatted{generic}({type} value, int alignment, string format)");
+                appendBody(includeValue: true, includeAlignment: true, includeFormat: true, isSpan);
+            }
+        }
+
+        [Theory]
+        [CombinatorialData]
+        public void InterpolatedStringBuilder_OverloadsAndBoolReturns(bool useDefaultParameters, bool useBoolReturns)
+        {
+            var source =
+@"int a = 1;
+System.Console.WriteLine($""base{a}{a,1}{a:X}{a,2:Y}"");";
+
+            string interpolatedStringBuilder = GetInterpolatedStringBuilderDefinition(includeSpanOverloads: false, useDefaultParameters, useBoolReturns);
+
+            string expectedOutput = useDefaultParameters ?
+@"base
+value:1,alignment:0:format:
+value:1,alignment:1:format:
+value:1,alignment:0:format:X
+value:1,alignment:2:format:Y" :
+@"base
+value:1
+value:1,alignment:1
+value:1:format:X
+value:1,alignment:2:format:Y";
+
+            string expectedIl = getIl();
+
+            var verifier = CompileAndVerify(new[] { source, interpolatedStringBuilder }, expectedOutput: expectedOutput);
+            verifier.VerifyIL("<top-level-statements-entry-point>", expectedIl);
+
+            var comp1 = CreateCompilation(interpolatedStringBuilder);
+
+            foreach (var reference in new[] { comp1.EmitToImageReference(), comp1.EmitToPortableExecutableReference() })
```

Comment:
>EmitToPortableExecutableReference [](http://example.com/codeflow?start=82&length=33)

It doesn't look like there is a difference by comparison to the first item. Did you mean to use ToMetadataReference?